src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.page {
+  --big-bar-height: 70px;
+  --big-bar-pleft: 62px;
+  --big-bar-border: 1px;
+  --big-search-icon-top: calc(var(--big-bar-height) / 2);
+
+  --small-bar-height: 42px;
+  --small-bar-pleft: 33px;
+  --small-bar-border: 1px;
+  --small-search-icon-top: calc(var(--small-bar-height) / 2);
+
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+/*
+.page-body {
+  margin: 0;
+  padding: 0;
+} */
+
+.container {
+  margin-top: 20px;
+  position: relative;
+  display: block;
+  height: var(--big-bar-height);
+}
+
+.container--small {
+  height: var(--small-bar-height);
+}
+
+.search-icon {
+  position: absolute;
+  top: var(--big-search-icon-top);
+  left: 26px;
+  transform: translateY(-50%);
+  width: 19px;
+  height: 19px;
+}
+
+.search-icon--small {
+  top: var(--small-search-icon-top);
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.bar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #fff;
```

Comment:
you do not really need position absolute here. it will naturally be in its place
bg-color is white by default as well

Reply:
I used this approach in the beginning. But the bg-color should become linear-gradient on focus, which overlays the background image of the icon.

Reply:
> you do not really need position absolute here. it will naturally be in its place bg-color is white by default as well

I used this approach in the beginning. But the bg-color should become linear-gradient on focus, which overlays the background image of the icon.

Reply:
I'm not sure what you mean.
I have disabled the positioning and the bg-color and it seems to work fine. 
![Screenshot 2022-03-15 at 10 42 48](https://user-images.githubusercontent.com/79282749/158340154-90b20a29-0eef-4f48-92bc-d80838e1b724.jpg)

![Screenshot 2022-03-15 at 10 42 13](https://user-images.githubusercontent.com/79282749/158339867-5e89813a-5799-493a-8e68-43e328bec1cd.jpg)


One thing I have noticed is that when I click on the big search bar, the shadow on the small one is altered